namer/consul/src/main/scala/io/buoyant/namer/consul/SvcAddr.scala

```diff
@@ -21,6 +22,8 @@ private[consul] object SvcAddr {
   private[this] val ServiceRelease =
     new Exception("service observation released") with NoStackTrace
 
+  implicit private[this] val timer: Timer = DefaultTimer
```

Comment:
This will be more testable if you pass the Timer in as a param into apply.  This allows us to pass in a MockTimer in tests.